src/ep.config.json

```diff
@@ -27,12 +25,12 @@
     }
   ],
   "defaultCurrencyValue": "CAD",
-  "skuImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%sku%.png",
-  "siteImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%fileName%",
+  "skuImagesUrl": "https://refstoreimgs.s3-us-west-2.amazonaws.com/productImages/%fileName%",
```

Comment:
Is this a copy of the existing bucket's images?

Reply:
Yes its a copy of existing buckets.  resized and re-typed versions of the breville catalog within the reference store aws